src/Api/SecretsManager/Models/Response/SMImportResponseModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Bit.Core.Models.Api;
+using Bit.Core.SecretsManager.Commands.Porting;
+
+namespace Bit.Api.SecretsManager.Models.Response;
+
+public class SMImportResponseModel : ResponseModel
+{
+    public SMImportResponseModel(SMImport import, string obj = "SecretsManagerImportResponseModel") : base(obj)
+    {
+        Projects = import.Projects != null && import.Projects.Any() ? import.Projects.Select(p => new InnerProject
+        {
+            Id = p.Id,
+            Name = p.Name,
+        }) : null;
```

Comment:
I believe this should work?

```suggestion
        Projects = import.Projects?.Select(p => new InnerProject
        {
            Id = p.Id,
            Name = p.Name,
        });
```

There is also a constructor for InnerProject, any reason we can't use that?

Reply:
Updated!